fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/email/exception/EmailConfigurationNotFoundException.java

```diff
@@ -28,4 +29,9 @@
     public EmailConfigurationNotFoundException(final String name) {
         super("error.msg.email.configuration.name.not.found", "Email configuration with name " + name + " does not exist", name);
```

Comment:
Just curious but wouldn't it be better to add the 3rd param here instead of duplicating the method? cause I can see the 2param method will just be history and unused.

Reply:
Nope, we cannot change the original implementation. If the cause of the exception is the exception itself, we need this constructor to be used.

Reply:
@thesmallstar @xurror the real question is really more if there are any (legitimate) remaining usages of the original constructor, without the cause. It seems like this was only used in `EmailConfigurationReadPlatformServiceImpl`, which was changed to use the new constructor, right? Then I actually agree that it would have been nicer and better to just add an additional parameter here... this isn't important enough to justify not finally merging this, so I'll probably go ahead, but I'll leave this open in case you'd like to clean this up in a new PR? (And perhaps in other exceptions where a cause was added - if there is no usage without a cause, it's better to simplify - and force future users to provide a cause.)

Reply:
Agreed! I will update this

Reply:
@vorburger  @xurror  making a new PR for this cleanup.